source/uwp/Renderer/idl/AdaptiveCards.Rendering.Uwp.idl

```diff
@@ -389,6 +389,15 @@ AdaptiveNamespaceStart
         All = Left | Right | Up | Down
     } BleedDirection;
 
+    [version(NTDDI_WIN10_RS1)]
+#ifdef ADAPTIVE_CARDS_WINDOWS
+    [contract(InternalContract, 1)][internal]
```

Comment:
not an issue with this PR in particular, but I thought `[internal]` was supposed to be on its own line? looks like we're inconsistent with it...

own line:
https://github.com/microsoft/AdaptiveCards/blob/58e35dbe6440432ed8c5d5253961d2734392cca1/source/uwp/Renderer/idl/AdaptiveCards.Rendering.Uwp.idl#L33-L45

just do whatever:
https://github.com/microsoft/AdaptiveCards/blob/58e35dbe6440432ed8c5d5253961d2734392cca1/source/uwp/Renderer/idl/AdaptiveCards.Rendering.Uwp.idl#L359-L367

Reply:
I'll move the ones that are not on their own line. For what it's worth, the autoformatting tries to move it to share the line, but it also makes a lot of other questionable changes. At some point we might want to figure out if there's a way to make the auto-formatting for .idl consistent with what we want (or just accept what it does so we're not fighting it).

---
In reply to: [537960705](https://github.com/microsoft/AdaptiveCards/pull/5141#discussion_r537960705) [](ancestors = 537960705)

Reply:
filed #5144 to track this -- thanks!